microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -123,7 +123,8 @@ Only one installation of the Evergreen WebView2 Runtime is needed for all Evergr
 
 *   For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
 
-Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation of permissions.  If an installer is run without elevation, the user is prompted to elevate permissions.
+> [!NOTE]
```

Comment:
global in article: check consistency of usage of phrases:
per-user
per-system
per-machine

Reply:
good catch, they are interchangeable and replaced all instances with per-machine

Reply:
Resolution/findings:
* per-user - 3 hits
* per-machine - 13 hits, per-system - 0 hits

So our lexicon in this article is: Use "per-user" & "per-machine".  Write "per-machine" instead of "per-system".